pkg/controllers/cloud/networkpolicy.go

```diff
@@ -109,6 +109,10 @@ const (
 	uniqueGroupNameMemberPrefix = "mm_"
 )
 
+const (
```

Comment:
nit: may be add this in other const block

Reply:
type is different, dont want to overload other block